templates/todo/web/react-fluent/package.json

```diff
@@ -65,6 +65,13 @@
     "async": "^3.2.3",
     "nth-check": "^2.0.1",
     "ejs": "^3.1.7",
-    "svgo": "^2.8.0"
+    "svgo": "^2.8.0",
+    "react-scripts":{
+      "react-dev-utils": {
+        "recursive-readdir": {
+          "minimatch": "3.0.5"
```

Comment:
Should we change this to `^3.0.5` so if there's another security update we can just regenerate the lock file?

Reply:
Done